flink-table/flink-table-common/src/test/java/org/apache/flink/table/expressions/ExpressionTest.java

```diff
@@ -160,6 +161,16 @@ public void testSymbolValueLiteralExtraction() {
 				.orElseThrow(AssertionError::new));
 	}
 
+	@Test
+	public void testPeriodlValueLiteralExtraction() {
```

Comment:
```suggestion
	public void testPeriodValueLiteralExtraction() {
```

Reply:
Thank you very much @wuchong. I have fix it.